src/test/lib/operators/get_table_test.cpp

```diff
@@ -357,4 +357,29 @@ TEST_F(OperatorsGetTableTest, AdaptOrderByInformation) {
   }
 }
 
+TEST_F(OperatorsGetTableTest, FinalizedChunks) {
+  // Insert one tuple into int_int_float to create a mutable chunk.
+  const auto& table = Hyrise::get().storage_manager.get_table("int_int_float");
+  EXPECT_EQ(table->chunk_count(), 4);
```

Comment:
Could we have the assumption somewhere that only the most recent chunk can be mutable? We could add a few more tuples to have multiple mutable chunks.

Do you know if we properly check that somewhere?

Reply:
Just added two mutable chunks here and check that the information is set correctly. Currently, I'm not sure where we should run into problems with that.